emulator/bundled/database_export/geohunt-1-default-rtdb.json

```diff
@@ -1,22 +1 @@
-{
-  "challenges": {
-    "163f921c": {
-      "-NQWln8MlqnVhArUIdwE": {
-        "authorId": "8b8b0392-ba8b-11ed-afa1-0242ac120002",
-        "expirationDate": "null",
-        "location": {
-          "latitude": 43.880433,
-          "longitude": -103.453748
-        },
-        "publishedDate": "2023-03-14T20:19:33.641982Z"
-      }
-    }
-  },
-  "users": {
-    "1": {
-      "score": 123,
-      "displayName": "Debug user",
-      "uid": "1"
-    }
-  }
-}
\ No newline at end of file
+{"challenges":{"163f921c":{"-NQWln8MlqnVhArUIdwE":{"authorId":"8b8b0392-ba8b-11ed-afa1-0242ac120002","expirationDate":"null","location":{"latitude":43.880433,"longitude":-103.453748},"publishedDate":"2023-03-14T20:19:33.641982Z"}}},"users":[null,{"displayName":"Debug user","score":123,"uid":"1"}]}
```

Comment:
Why is "users" now an array?

And isn't possible to keep a beautif-ied json? I think it's better for everyone.

Reply:
Note that using arrays in firebase is not a good idea.

Reply:
This is a generated file don't ask question. You can generate it by adding the option `--export-on-exit` to the emulator. I have no idea why it did that. Guess it was an version update

Reply:
Still, I think that it should be better if any modification is made "correctly", like a human written JSON. Not something that can be overwritten by ANY changes made on a firebase cli emulator instance.

Reply:
Do you also want to comment on the blob file generated by this the